File: Raven.Tests/Linq/CommitInfo.cs
//-----------------------------------------------------------------------
// <copyright file="CommitInfo.cs" company="Hibernating Rhinos LTD">
//     Copyright (c) Hibernating Rhinos LTD. All rights reserved.
// </copyright>
//-----------------------------------------------------------------------
using System;

namespace Raven.Tests.Linq
{
	public class CommitInfo
	{
		public string Id { get; set; }
		public string Author { get; set; }
		public string PathInRepo { get; set; }
		public string Repository { get; set; }
		public int Revision { get; set; }
		public DateTime Date { get; set; }
		public string CommitMessage { get; set; }
	}
}

File: Raven.Client.Java/src/test/java/net/ravendb/tests/linq/CommitInfo.java
package net.ravendb.tests.linq;

import java.util.Date;

import com.mysema.query.annotations.QueryEntity;

@QueryEntity
public class CommitInfo {

  private String id;
  private String author;
  private String pathInRepo;
  private String repository;
  private int revision;
  private Date date;
  private String commitMessage;
  public String getId() {
    return id;
  }
  public void setId(String id) {
    this.id = id;
  }
  public String getAuthor() {
    return author;
  }
  public void setAuthor(String author) {
    this.author = author;
  }
  public String getPathInRepo() {
    return pathInRepo;
  }
  public void setPathInRepo(String pathInRepo) {
    this.pathInRepo = pathInRepo;
  }
  public String getRepository() {
    return repository;
  }
  public void setRepository(String repository) {
    this.repository = repository;
  }
  public int getRevision() {
    return revision;
  }
  public void setRevision(int revision) {
    this.revision = revision;
  }
  public Date getDate() {
    return date;
  }
  public void setDate(Date date) {
    this.date = date;
  }
  public String getCommitMessage() {
    return commitMessage;
  }
  public void setCommitMessage(String commitMessage) {
    this.commitMessage = commitMessage;
  }


}
